.github/troubleshooting.md

```diff
@@ -29,8 +29,7 @@ In case of errors in any of these steps, the log should be inspected to
 check what was the last action the pipeline executed. The pipelines are not idempotent so to retry them if there was a failure in the release step, manual amends should be done to prepare the next run:
 
 * Delete the tags and the release on GitHub (if exists)
-* Revert the commits made by the pipeline on the version branch (if there are commits made by kiali-bot on the same time around the pipeline run)
-* Delete the PR generated if exists
```

Comment:
@leandroberetta you wrote this troubleshooting doc last year. Can you review this? I don't think these lines should be outright deleted. I think just mention of kiali-bot should be removed. However, we do still generate commits and PRs from GitHub Actions that need to be reverted in the case of manual intervention, which is what this part of the docs are talking about.

Also, please review the other two changes in here.

Reply:
It's true that the kiali-bot account is gone, but, the gh actions requires to configure the user and email (check https://github.com/kiali/kiali/blob/master/.github/workflows/release.yml#L246), I'm not sure if today there is a better way to do this, in some post back in that time, this was needed. And the kiali-bot works (really, whatever there will work) so there is still a commit by kiali-bot:

![image](https://github.com/kiali/kiali/assets/1286393/88bff96d-3cb6-4cc4-9420-6a6a950df6be)

So, right, it doesn't need to be removed, the github-actions[bot] user also is part of the commit, so I think this requires an investigation to see if we can remove the kiali-bot configuration in our actions, but whoever does the commit, if there is any error, the commit should be reverted.